paddle/fluid/operators/abs_op.cc

```diff
@@ -156,6 +177,7 @@ REGISTER_OPERATOR(abs,
 
 REGISTER_OPERATOR(abs_grad,
                   ops::AbsGradOp,
+                  ops::AbsCompositeGradOpMaker,
```

Comment:
Should register this into abs instead of abs_grad

Reply:
fixed